core/src/templates/components/card/card.twig

```diff
@@ -133,13 +143,14 @@
     {% endif %}
   {% endblock block_card_media -%}
 
-  <section class="su-card__contents">
+  {%- if card_contents_not_empty -%}
```

Comment:
Early on we asked if a card should ever be if it doesn't have a body. The general answer was "no" if the card does not have a body it is not a card. What is the use case for this pattern?

Reply:
@sherakama 
In https://stanfordits.atlassian.net/browse/D8CORE-2151, Ishi recommended removing the empty space. This seemed to be the most appropriate way of removing this space.

Please advise.

Reply:
Implement field validation on the card paragraph to require one of the values for the body area in cards and use the media paragraph for when you only want to display an image or video.